draft-ietf-ppm-dap.md

```diff
@@ -485,7 +485,7 @@ In addition, in order to facilitate the aggregation and collect protocols, each
 of the aggregators is configured with following parameters:
 
 * `collector_config`: The HPKE configuration of the collector (described in
-  {{key-config}}).
+  {{hpke-config}}).
```

Comment:
This referenced section is by and large particular to the client requesting an HPKE configuration from the leader, but it does include relevant structure definitions. How's this?

```
... The HPKE configuration of the collector (`HpkeConfig` defined in {{hpke-config}}).
```

Reply:
IMO we should fix this by moving the definition of `struct HpkeConfig` and its attendant types from `{{hpke-config}}` up to the common type definitions in "Protocol Definition" to eliminate this awkward forward reference. I'll do that in a separate change.